### PR TITLE
Fixup tests to run under cpython ok.

### PR DESCRIPTION
**Pull request by bbangert**

### CODE REVIEW

File: autopush/tests/test_bridge.py [hunk 1]
```diff
@@ -1,7 +1,4 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
+# -*- coding: utf-8 -*-
```
Comment (jrconlin) [inline autopush/tests/test_bridge.py]: Why are we stripping the MPL?

File: autopush/tests/test_endpoint.py [hunk 2]
```diff
@@ -770,10 +770,13 @@ def test_get_valid(self, arg):
 
         def handle_finish(value):
             self.reg.set_status.assert_called_with(200)
-            self.reg.write.assert_called_with(
-                '{"useragentid": "' + dummy_uaid + '", '
-                '"channelid": "' + dummy_chid + '", '
-                '"endpoint": "http://localhost/push/abcd123"}')
+            called_arg = dict(useragentid=dummy_uaid,
```
Comment (kitcambridge) [inline autopush/tests/test_endpoint.py]: I didn't realize this notation existed. :heart:

File: autopush/tests/test_endpoint.py [hunk 3]
```diff
@@ -770,10 +770,13 @@ def test_get_valid(self, arg):
...
+                              channelid=dummy_chid,
+                              endpoint="http://localhost/push/abcd123")
+            call_args = self.reg.write.call_args
+            ok_(call_args is not None)
+            args = call_args[0]
+            eq_(eval(args[0]), called_arg)
```
Comment (kitcambridge) [inline autopush/tests/test_endpoint.py]: :smile_cat:

File: autopush/tests/test_bridge.py (hunk 1 above)
Comment (bbangert) [inline autopush/tests/test_bridge.py]: Why does no other file in the project have it? The LICENSE file already has the MPL, why does it need to start appearing now? (Oh hey, now that I look, I see we don't have a LICENSE file, one sec, adding it to this PR)

File: autopush/tests/test_endpoint.py (hunk 2 above)
Comment (jrconlin) [inline autopush/tests/test_endpoint.py]: also a new fan.

File: autopush/tests/test_bridge.py (hunk 1 above)
Comment (bbangert) [inline autopush/tests/test_bridge.py]: License file added.

File: autopush/tests/test_endpoint.py (hunk 3 above)
Comment (bbangert) [inline autopush/tests/test_endpoint.py]: I feel gross inside for using eval, but its the easiest way to turn the str'd dict back into a dict for proper comparison. :disappointed:

File: autopush/tests/test_bridge.py (hunk 1 above)
Comment (jrconlin) [inline autopush/tests/test_bridge.py]: Ah, I have a vim template file that automatically adds it.

File: autopush/tests/test_endpoint.py (hunk 3 above)
Comment (jrconlin) [inline autopush/tests/test_endpoint.py]: could we cheat and call a JSON for this? it might be a tiny bit safer.

**Comment by jrconlin** [comment]
r+

**Pull request merged**